### ISSUE TITLE
Step 2 Wring lambda

### ISSUE THREAD

**Issue by ronki2304** [alexa/skill-sample-nodejs-city-guide#37]
the step 6 is "Search for the application repository named "alexa-skills-kit-nodejs-factskill". You can find it using the provided search box. "

but factskill is for the first tutorial, and I can't find the city guide lambda package.

**Comment by mikevocalz** [comment]
im having the same issue

**Comment by triplel** [comment]
Hey guys, you will need to replace the content of the index.js with the https://github.com/alexa/skill-sample-nodejs-city-guide/blob/master/lambda/custom/index.js in this repository. 

Once you finished the copy over and saved, you should be run unit test described in Step 4. If you see error related to the project cannot find i18next or -sprintf-postprocessor, the best/quickest fix is to add those files yourselves, here are two sources of the js modules that are missing:
https://github.com/i18next/i18next/blob/master/i18next.js
https://github.com/i18next/i18next-sprintf-postProcessor/blob/master/i18nextSprintfPostProcessor.js

Basically, you can think of alexa-skills-kit-nodejs-factskill as a boiler plate skill template for all fact based skills. You can replace it with your one lambda logic and datasets that works with your Alexa skill.

**Comment by snobordir** [comment]
Thank you triplel, your comment gave me info to get my code testing correctly. 

My question is, why did I have to dig into one of the issues to find this info? I know it's an "advanced" tutorial, but I don't think that should include having to find obscure code in a GitHub issue and then figure out how to add it. My only indication of a problem was the Alexa Simulator saying "there was a problem with the requested skill's response." I had to plug the JSON input into the lambda manager test event thing to know what was really going on.

For anyone else reading this: I created two new .js files and added them into the "Function Code" section of the Lambda management console. Got the code that triplel mentions above. Named the .js files after what the code was looking for (the 'require' constants near the top of index.js).

This tutorial isn't much of a tutorial at all, it's the fact skill tutorial with a couple links changed. It has a bug built in with the i18next thing.
